src/contracts/facilitators/gsm/feeStrategy/interfaces/IFixedFeeStrategyFactory.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IFixedFeeStrategyFactory {
```

Comment:
missing docs

Reply:
will add. Flagging that `interface IFixedRateStrategyFactory` is also missing docs

Reply:
Done